modules/geopackage/src/lib/parse-geopackage.ts

```diff
@@ -15,7 +15,7 @@ import {
   Float32,
   Binary
 } from '@loaders.gl/schema';
-import {binaryToGeoJson, transformGeoJsonCoords} from '@loaders.gl/gis';
+import {binaryToGeoJSON, transformGeoJsonCoords} from '@loaders.gl/gis';
```

Comment:
Let's be consistent with GeoJson capitalization. We shouldn't change it in just one place, suggest leaving it lowercase (i.e the capitalizated form of `geojson` in `geojsonToBinary`)

Reply:
Well it was not consistent with `geojsonToBinary` before. I can rename it `binaryToGeojson` (lower case J) instead.

Reply:
Yes, that sounds good. 

Agree that the naming is a bit of a mess. ideally if we make a bigger change we should do it across the board.

Reply:
Renamed to binaryToGeojson